components/ModEditSubmissionTopbar.vue

```diff
@@ -22,11 +22,30 @@
             </button>
         </div>
         <div class="flex flex-row justify p-2 font-bold ">
+            <p
+                v-show="showRetryMessage"
+                class="flex flex-row justify-center items-center whitespace-nowrap text-error text-xs mr-2"
+                data-testid="submission-topNav-retryMessage"
+            >
+                <SVGCautionSign class="w-3 h-3 mr-1" /> {{ retryMessage }}
+            </p>
             <button
                 class="flex justify-center items-center rounded-full bg-secondary-bg border-primary-text-muted
-                border-2 w-28 text-sm mr-2 "
+                border-2 w-28 text-sm mr-2"
+                data-testid="submission-topNav-saveAndExit"
+                @click="saveAndExit"
             >
-                {{ $t('modEditSubmissionTopNav.saveAndExit') }}
+                <span
+                    v-show="updatedFormBeingSentForMutation"
```

Comment:
This shows saving text. Very briefly but follows the decision from Figma. If the request takes a while it will show. Depends on speed of server response.